docs/content/src/learn/guides/database-integrations/mongo-db-transactions/insert-mongodb/README.md

```diff
@@ -0,0 +1,99 @@
+# Importing CSV Data to Mongo DB
+
+## About 
+Ballerina is an open-source programming language that empowers developers to integrate their system easily with the support of connectors. In this guide, we are mainly focusing on importing CSV file having contacts into MongoDB using FTP connector.
+
+`wso2/mongodb` module allows you to perform CRUD operations on Mongo DB.<br/> 
+The `wso2/ftp` module provides an FTP client and an FTP server listener implementation to facilitate an FTP connection 
+to a remote location. You can find other integration modules from the [wso2-ballerina](https://github.com/wso2-ballerina) Github repository. 
+
+## What you'll build
+
+This application listens to a remote FTP location and when a CSV file is added to that FTP location, it will fetch the CSV file, read its contents and insert the content into Mongo DB. Then a 
+success message is logged if the operation is successful.
+
+![inserting csv data to mongo db](../../../../../assets/img/mongo_insert.png)
+
+## Prerequisites
+Link to download Ballerina integrator.
+
+## Implementation
+The Ballerina project should create for the integration use case explained above. Please follow the steps given below. You can learn about the Ballerina project and module by following the [guide](https://ei.docs.wso2.com/en/latest/ballerina-integrator/develop/using-modules/).
```

Comment:
Let's make this a relative link. So it will be something like: (../../develop/using-modules/).

Reply:
Fixed